### DISCUSSION TITLE
Tech Talk 7 Contributions

### DISCUSSION THREAD

**Discussion by timokonkwo** [timokonkwo/techtalk#45] [General]
## What's Changed
* Tech Talk 6.0 Page Submission by @PrevailExcel in https://github.com/timokonkwo/techtalk/pull/2
* Update README.md by @timokonkwo in https://github.com/timokonkwo/techtalk/pull/3
* Updated README by @Onyekachukwu-Nweke in https://github.com/timokonkwo/techtalk/pull/5
* Chosen landing page for Tech Talk 7.0 by @timokonkwo in https://github.com/timokonkwo/techtalk/pull/8
* Change all instances of Tech Talk 6.0 to 7.0 by @HyperNova114 in https://github.com/timokonkwo/techtalk/pull/12
* added alt attribute to the images to aid screen readers. by @starkweb108 in https://github.com/timokonkwo/techtalk/pull/13
* just a littel punctuation correction by @ken286 in https://github.com/timokonkwo/techtalk/pull/16
* fix alignmemt of the speakers pictures by @fadouaabdoul in https://github.com/timokonkwo/techtalk/pull/17
* Making the Company to be more visible to the Viewers by @SardiusJay in https://github.com/timokonkwo/techtalk/pull/18
* I changed the main color of the boxes titles (h4) | Added the title property to all navbar links. by @boularbahsmail in https://github.com/timokonkwo/techtalk/pull/21
* minor spelling fix by @dapoadedire in https://github.com/timokonkwo/techtalk/pull/31
* Fix form validation for phone number issues #29 by @ndkhangvl in https://github.com/timokonkwo/techtalk/pull/32
* Updated the readme by @Chukwuneme in https://github.com/timokonkwo/techtalk/pull/36
* Updated links to code of conduct.md and contributing.md by @Chukwuneme in https://github.com/timokonkwo/techtalk/pull/43
* feat: added social links. by @devBayo in https://github.com/timokonkwo/techtalk/pull/44

## New Contributors
* @PrevailExcel made their first contribution in https://github.com/timokonkwo/techtalk/pull/2
* @timokonkwo made their first contribution in https://github.com/timokonkwo/techtalk/pull/3
* @Onyekachukwu-Nweke made their first contribution in https://github.com/timokonkwo/techtalk/pull/5
* @HyperNova114 made their first contribution in https://github.com/timokonkwo/techtalk/pull/12
* @starkweb108 made their first contribution in https://github.com/timokonkwo/techtalk/pull/13
* @ken286 made their first contribution in https://github.com/timokonkwo/techtalk/pull/16
* @fadouaabdoul made their first contribution in https://github.com/timokonkwo/techtalk/pull/17
* @SardiusJay made their first contribution in https://github.com/timokonkwo/techtalk/pull/18
* @boularbahsmail made their first contribution in https://github.com/timokonkwo/techtalk/pull/21
* @dapoadedire made their first contribution in https://github.com/timokonkwo/techtalk/pull/31
* @ndkhangvl made their first contribution in https://github.com/timokonkwo/techtalk/pull/32
* @Chukwuneme made their first contribution in https://github.com/timokonkwo/techtalk/pull/36
* @devBayo made their first contribution in https://github.com/timokonkwo/techtalk/pull/44

**Full Changelog**: https://github.com/timokonkwo/techtalk/commits/v0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/timokonkwo/techtalk/releases/tag/v0.2'>Tech Talk 7 Contributions</a>.</em>

**Comment by ajSeadler** [comment]
Is this repo still active?
